actionview/lib/action_view/layouts.rb

```diff
@@ -411,9 +411,9 @@ def _normalize_layout(value)
     #
     # ==== Returns
     # * <tt>template</tt> - The template object for the default layout (or +nil+)
-    def _default_layout(formats, require_layout = false)
+    def _default_layout(lookup_context, formats, require_layout = false)
```

Comment:
@tenderlove I think the comment above needs to be updated to talk about the new argument.